src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -170,6 +195,24 @@ private void handleExit() {
         primaryStage.hide();
     }
 
+    /**
+     * Display and refresh date time display on TutorBuddy every 1 second
+     */
+    private void displayDateAndTime() {
+        final Timeline timeline = new Timeline(
+                new KeyFrame(
+                    Duration.millis(1000),
+                    event -> {
+                        time.setText(LocalDateTime.now().toLocalTime().format(DateTimeFormatter.ofPattern("HH:mm"))
+                            + "     " + LocalDateTime.now().toLocalDate()
+                                .format(DateTimeFormatter.ofPattern("dd-MM-YYYY")));
```

Comment:
Perhaps would be better if `dd MMM YYYY`, so it'll be 29 Mar 2021. more reader friendly!

Reply:
Agreed, `dd MMM YYYY` would be more reader friendly.

Reply:
Ok updated!